blocks/api/raw-handling/test/integration/index.js

```diff
@@ -14,6 +14,7 @@ import serialize from '../../../serializer';
 
 const types = [
 	'plain',
+	'classic',
```

Comment:
I wonder if we could consolidate the other image and iframe test into one "classic" conversion test similar to the other integration tests.

Reply:
I'd like that, or something along those lines. Better left to another PR, though, no?